ultralytics/engine/exporter.py

```diff
@@ -138,7 +138,7 @@ def outer_func(*args, **kwargs):
             LOGGER.info(f"{prefix} export success ✅ {dt.t:.1f}s, saved as '{f}' ({file_size(f):.1f} MB)")
             return f, model
         except Exception as e:
-            LOGGER.info(f"{prefix} export failure ❌ {dt.t:.1f}s: {e}")
+            LOGGER.fatal(f"{prefix} FATAL ⚠️ export failure ❌ {dt.t:.1f}s: {e}")
```

Comment:
I think one emoji per line suffices. We use the yellow triangle for warnings and the red X for errors, so I think bumping from .info() to .error() is appropriate here, but we don't need the fatal() as it's not used anywhere else in the repo and this exception handler is intended for processes to survive these errors.